package.json

```diff
@@ -48,7 +48,7 @@
     "eslint-plugin-mocha": "5.3.0",
     "eslint-plugin-new-with-error": "2.0.0",
     "eslint-plugin-react": "7.14.3",
-    "eslint-plugin-react-hooks": "1.6.1",
+    "eslint-plugin-react-hooks": "^4.5.0",
```

Comment:
Please use a fixed version here. Using a `^` range here will allow installations to use newer version, which historically cause problems with eslint plugins.